app/helpers/utils.py

```diff
@@ -36,3 +44,47 @@ def get_logging_cfg():
 def init_logging():
     config = get_logging_cfg()
     logging.config.dictConfig(config)
+
+
+def prevent_erroneous_kml(kml_string):
+    kml_string = re.sub(r'on\w*=(".+?"|\'.+?\')', '', kml_string, flags=re.IGNORECASE)
```

Comment:
A small docstring that explain what this function does, e.g. remove all attributes with `on` prefix and remove all `script` elements.